boards/arm/qemu_cortex_a53/board.cmake

```diff
@@ -5,9 +5,17 @@ set(EMU_PLATFORM qemu)
 set(QEMU_ARCH aarch64)
 
 set(QEMU_CPU_TYPE_${ARCH} cortex-a53)
+if("${CONFIG_ARMV8_A_NS}" STREQUAL "y")
+set(QEMU_FLAGS_${ARCH}
+  -cpu ${QEMU_CPU_TYPE_${ARCH}}
+  -nographic
+  -machine virt,gic-version=3
+  )
+else()
```

Comment:
No need to use `STREQUAL` with `CONFIG_*` boolean symbols. You can just use `if(CONFIG_ARMV8_A_NS)...`.

Also instead of duplicating all the options can you just introduce `QEMU_EXTRA_FLAGS` and use that to differentiate between the two cases?